backend/backend/roll_two_dice.py

```diff
@@ -0,0 +1,18 @@
+""" Module to provide functionality to simulate rolling of two 6-sided dice"""
+
+from backend.roll_die import roll_dice
+
+
+def roll_two_dice():
+    """Simulates the rolling of two 6-sided dice.
+
+    Returns:
+        An int array representing the result of the two dice rolls.
```

Comment:
Could we do a tuple rather than a list?

Reply:
I was unsure about how how JavaScript would interpret tuples - if it ever had to be passed to some JavaScript

Reply:
When it gets passed to JSON Python will convert it to a list